Tasks/Gradle/CodeAnalysis/SonarQube/run-settings.ts

```diff
@@ -50,7 +50,8 @@ export class SonarQubeRunSettings {
      * @returns {SonarQubeRunSettings} A new SonarQubeRunSettings with appropriate fields filled
      */
     public static createRunSettingsFromFile(filePath:string): SonarQubeRunSettings {
-        if (!tl.exist(filePath)) {
+        
+        if (!fs.existsSync(filePath)) {
```

Comment:
This function is deprecated. See the following:

https://nodejs.org/api/fs.html#fs_fs_existssync_path
http://stackoverflow.com/questions/28532820/fs-exists-fs-existssync-why-are-they-deprecated